### PR TITLE
Simple demo

### PR DESCRIPTION
**Pull request by mlautman**
This PR adds a simple demo to the moveit_simple project so that developers can visually inspect the system after making changes without having to rebuild all of the tests which takes approximately 4 minutes. This PR builds off of the Readme changes in #84

### CODE REVIEW

**Comment by mlautman** [comment]
This should target a master/devel branch

**Comment by mlautman** [comment]
@aaronplusone @henningkayser

File: moveit_simple/scripts/trajectory_planner_demo.cpp [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/*
+ * Software License Agreement (Apache License)
+ *
+ * Copyright (c) 2016 Shaun Edwards
+ * Copyright (c) 2018 Plus One Robotics
```
Comment (henningkayser) [inline moveit_simple/scripts/trajectory_planner_demo.cpp]: Update dates

File: README.md [hunk 2]
```diff
@@ -66,3 +66,16 @@ rostest moveit_simple motoman_mh5_utest.launch -r
...
+
+## Quick Demo
+
+1. Launch the RViz visualizer
+```
+roslaunch moveit_simple test_display.launch
+```
+
+1. Launch the demo
+```
```
Comment (henningkayser) [inline README.md:113]: I think `test_display.launch` should be included inside `kuka_kr210_demo.launch` so the robot model panel doesn't need to be refreshed. Also is there a reason not to change the rviz config to use `base_link` as fixed frame by default?

Comment (mlautman) [reply README.md:113]: The motoman has a world link

**Comment by shaun-edwards** [comment]
Does this have any conflicts with changes made to the README, here: https://github.com/plusone-robotics/moveit_simple/pull/84

**Comment by mlautman** [comment]
@shaun-edwards It shouldn't be a problem. Once we've merged #84 I will resolve all conflicts

**Pull request merged**